src/routes/generator/Toolbar.svelte

```diff
@@ -1,51 +1,120 @@
 <script lang="ts">
 	import { isMapModalOpen, userProvidedLink } from './stores';
+	import Save from '../../icons/Save.svg';
+	import Redo from '../../icons/Redo.svg';
 	let nameOfMap = '';
 </script>
 
 <div class="buttonContainer">
```

Comment:
You might want to be consistent in naming of this, so this class should be toolbar (or just container)